api/routes/apds/get.js

```diff
@@ -29,4 +29,34 @@ module.exports = (app, ApdModel = defaultApdModel) => {
       return res.status(500).end();
     }
   });
+
+  app.get('/apds/:id(\\d+)', can('view-document'), async (req, res) => {
```

Comment:
Makes this path match more precise with a regex.  Otherwise it collides with `/apds/submitted`, and the winner is whichever one just happens to be called last.   If this one is called, then it thinks the APD ID is `submitted`, which blows up the database.  But hey, we can avoid that by just making this more precise.